src/Tasks/GenerateResource.cs

```diff
@@ -2895,14 +2895,14 @@ private string EnsurePathIsShortEnough(string currentOutputFile, string currentO
         /// <param name="filename">Full path to file to delete</param>
         private void RemoveCorruptedFile(string filename)
         {
-            _logger.LogWarningWithCodeFromResources("GenerateResource.CorruptOutput", FileUtilities.GetFullPathNoThrow(filename));
+            _logger.LogMessageFromResources("GenerateResource.CorruptOutput. Attempting to remove possibly-corrupt output file: ", FileUtilities.GetFullPathNoThrow(filename));
```

Comment:
This is changing the name of the resource, and you shouldn't need to do that.
```suggestion
            _logger.LogMessageFromResources("GenerateResource.CorruptOutput", FileUtilities.GetFullPathNoThrow(filename));
```

Reply:
I added that in because of this:

> The warning seems unnecessary. We should bump it down to info and make it more descriptive, maybe something like
>
>   ` Attempting to remove possibly-corrupt output file {}.`

How should I work that in then?

Reply:
Change the contents of the string (in the `.resx` file and localizations in the various `.xlf`), not its name.

As an analogy, you're doing this:

```diff
- var generateResource_CorruptOutput = "MSB3555: Output file "{0}" is possibly corrupt.";
+ var generateResource_CorruptOutput_attempting_to_remove = "MSB3555: Output file "{0}" is possibly corrupt.";
```

instead of

```diff
- var generateResource_CorruptOutput = "MSB3555: Output file "{0}" is possibly corrupt.";
+ var generateResource_CorruptOutput = "Output file "{0}" is possibly corrupt.";
```